lang/tags/operators_math.talon

```diff
@@ -0,0 +1,25 @@
+tag: user.code_operators_math
+-
+
+# math operators
+op (minus | subtract): user.code_operator_subtraction()
+op (plus | add): user.code_operator_addition()
+op (times | multiply): user.code_operator_multiplication()
+op divide: user.code_operator_division()
+op mod: user.code_operator_modulo()
+(op (power | exponent) | to the power [of]): user.code_operator_exponent()
+
+# comparison operators
+(op | is) equal: user.code_operator_equal()
+(op | is) not equal: user.code_operator_not_equal()
+(op | is) (greater | more): user.code_operator_greater_than()
+(op | is) (less | below) [than]: user.code_operator_less_than()
+(op | is) greater [than] or equal: user.code_operator_greater_than_or_equal_to()
+(op | is) less [than] or equal: user.code_operator_less_than_or_equal_to()
+
+# logical operators
+(op | logical) and: user.code_operator_and()
+(op | logical) or: user.code_operator_or()
+
+# TODO: This operator should either be abstracted into a function or removed.
+(op | pad) colon: " : "
```

Comment:
I imagine this is commonly used in C++ and Java, where it's used in for-each loops, but also subclassing in C++. There's probably a better place to put this than lumping them into the maths operators.

Reply:
It's not really an abstracted operator, who am not exactly sure where it should go. Ideally we would abstract it out and give it some sensible name. Right now it has no semantics associated with it, and simply print a spaced colon.

Reply:
From what you have said it should be split up into several different operators, one for subclassing and one for "in" in foreach loops.

Reply:
For now I can move it to a miscellaneous tag, since deciding what should happen to this one command is beyond the scope of this pull request.

Reply:
I think it's probably better to just leave this where it is than to create a whole file for one operator.

I also don't think operators/commands need to have semantic names; just memorable ones.

Reply:
@auscompgeek I have created a miscellaneous tag as requested.